SharpPcap/SharpPcap.csproj

```diff
@@ -33,8 +33,6 @@ SPDX-License-Identifier: MIT
     <PackageReference Include="System.Memory" Version="4.5.5" />
     <PackageReference Include="System.Text.Encoding.CodePages" Version="6.0.0" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0" PrivateAssets="All"/>
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0" PrivateAssets="All"/>
-    <PackageReference Include="System.Text.Encoding.CodePages" Version="8.0.0" />
```

Comment:
keep the 8.0.0 version, and delete the 6.0.0 one

Reply:
Ah yes, apologies! Too quick with the deletion 😄